salt/pillar/journal.sls

```diff
@@ -68,6 +68,10 @@ journal:
 
     cache_control: private, no-cache, no-store, must-revalidate
 
+    feature_rds: true
+    # 16846 is in api-dummy
+    rds_articles:
+        '16846': https://hub.stenci.la/elife/30274/main/
```

Comment:
```suggestion
        '16846':
            display: https://hub.stenci.la/elife/30274/main/
```

Reply:
Oh yeah, missed that. I guess with the current `master` there's no test that could find out, thanks